### PR TITLE
Munch task87

### PR DESCRIPTION
**Pull request by xulean**
added 126 images

### CODE REVIEW

**Comment by xulean** [comment]
please review it

File: backend/.gitignore
Comment (kelseyp) [inline backend/.gitignore:1]: Please do not delete the .gitignore

File: frontend/.gitignore
Comment (kelseyp) [inline frontend/.gitignore:1]: Please do not delete the .gitignore

File: backend/package-lock.json
Comment (kelseyp) [inline backend/package-lock.json:1]: There should be no reason to delete the package-lock.json

File: frontend/package-lock.json
Comment (kelseyp) [inline frontend/package-lock.json:1]: There should be no reason to delete the package-lock.json

**Review by kelseyp — DISMISSED** [review]

**Review by austinhedrick548 — APPROVED** [review]
Works good, just doesn't load in codespaces for some reason

**Review by NathanaelPaulus — APPROVED** [review]
looks good

**Pull request merged**